zeus.nix

```diff
@@ -1 +0,0 @@
-(import ./. { runTests = false; })
```

Comment:
Do we want to remove zeus? is it used?

Reply:
It used to be used by Zeus as the entry point, but Zeus can't build chainweb-node with the new flake + haskell.nix infrastructure due to it having an old Nix version. In any case, we now have a github action for building and caching the results, so Zeus is obsoleted as far as chainweb-node is concerned.